src/lib/autocomplete/autocomplete.spec.ts

```diff
@@ -853,6 +854,84 @@ describe('MdAutocomplete', () => {
 
   });
 
+  describe('option groups', () => {
+    let fixture: ComponentFixture<AutocompleteWithGroups>;
+    let DOWN_ARROW_EVENT: KeyboardEvent;
+    let UP_ARROW_EVENT: KeyboardEvent;
+
+    beforeEach(() => {
+      fixture = TestBed.createComponent(AutocompleteWithGroups);
+      fixture.detectChanges();
+
+      DOWN_ARROW_EVENT = createKeyboardEvent('keydown', DOWN_ARROW);
+      UP_ARROW_EVENT = createKeyboardEvent('keydown', UP_ARROW);
+
+      fixture.componentInstance.trigger.openPanel();
+      fixture.detectChanges();
+    });
+
+    it('should scroll to active options below the fold', fakeAsync(() => {
+      tick();
+      const container = document.querySelector('.cdk-overlay-pane .mat-autocomplete-panel')!;
```

Comment:
How about querying that element in the `beforeEach`? The `tick()` and `container` variable initialization is done in every test of the current `describe`.